src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,28 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  return (
+    (completed
+      ? (
```

Comment:
i see difference only in className, so move this condition in className. Not needed to add 2 options with no difference :)